### ISSUE TITLE
Publishing takes forever sometimes

### ISSUE THREAD

**Issue by mAAdhaTTah** [mAAdhaTTah/wordpress-github-sync#101]
I suspect this has to do w/ the number of API calls we're making. I suspect the current caching setup and be spruced up. This should probably be resolved as part of the massive refactor going on in #98.

**Comment by mAAdhaTTah** [comment]
So we're still having issues with the objects being too big (#72) when transients are saving to the database instead of the object cache. Or at least I am, so maybe we just want to bypass the database and go straight to object cache always? Or maybe base64encode the serialized string?

**Comment by mAAdhaTTah** [comment]
It definitely feels like it's gotten faster though; a lot of the blobs are being correctly cached, so we're definitely saving some time there. Maybe that's a way forward? Cache blobs in transients, trees/commits in object cache?

**Comment by mAAdhaTTah** [comment]
This was actually a problem with my repository, which used to have [this](https://github.com/mAAdhaTTah/writing/blob/2fe5f2c7c31e0bbfbb22665e8654049f2f6795c6/_drafts/Building%20a%20Hacktintosh%20-%20Background%20Parts%20and%20Assembly.md) in it, which you'll notice is 2.2MBs. That's a bunch of inline images, base64-encoded, that I got when I converted a Word document to markdown with [this gem](https://github.com/benbalter/word-to-markdown). The cache works perfectly once that was removed.

My thought is that any site that approaches that size should be using an object cache, so we won't run into that issue as an actual problem during anyone's use. I also found out that my object cache was improperly configured, so this led me to resolve that as well.
